pipeline/cloud-functions/node/utils/airqo-insert-one.js

```diff
@@ -0,0 +1,687 @@
+const axios = require("axios");
+const { parentPort, workerData, isMainThread } = require("worker_threads");
```

Comment:
Hi @Priscilla-AL 
This is the main script that was modified and used in the cloud function (**get_airqo_device_measurements**).
This PR is just attempting to reorganise this code into endpoints....


I guess we shall just go this during the PR review meeting since I am still changing the structure of the codebase.
I am also considering moving this code to data-mgt service since it is a natural fit.

Reply:
Okay, noted